### ISSUE TITLE
github workflows build failed

### ISSUE THREAD

**Issue by umjammer** [umjammer/vavi-net-auth#2]
* javafx
* jitpack is ok

https://github.com/umjammer/vavi-net-auth/runs/651563799

**Comment by umjammer** [comment]
jitpack
```
Picked up JAVA_TOOL_OPTIONS: -Dfile.encoding=UTF-8 -Dhttps.protocols=TLSv1.2
Apache Maven 3.6.1 (d66c9c0b3152b2e69ee9bac180bb8fcc8e6af555; 2019-04-04T19:00:29Z)
Maven home: /opt/apache-maven-3.6.1
Java version: 1.8.0_252, vendor: Private Build, runtime: /usr/lib/jvm/java-8-openjdk-amd64/jre
Default locale: en, platform encoding: UTF-8
OS name: "linux", version: "4.10.0-28-generic", arch: "amd64", family: "unix"

Running: mvn clean install -DskipTests
```

**Comment by umjammer** [comment]
github
```
Operating System
  macOS
  11.6.4

java-version: 8
    java-package: jdk
    architecture: x64
    server-id: github

Run mvn -B install --file pom.xml -DskipTests
  mvn -B install --file pom.xml -DskipTests
  shell: /bin/bash -e {0}
  env:

JAVA_HOME_8.0.322_x64: /Users/runner/hostedtoolcache/jdk/8.0.322/x64
    JAVA_HOME: /Users/runner/hostedtoolcache/jdk/8.0.322/x64
    JAVA_HOME_8_0_322_X64: /Users/runner/hostedtoolcache/jdk/8.0.322/x64
```

**Comment by umjammer** [comment]
fixed in 49f1c55
